tomcat/src/main/java/org/apache/catalina/controller/LoginController.java

```diff
@@ -0,0 +1,69 @@
+package org.apache.catalina.controller;
+
+import com.techcourse.db.InMemoryUserRepository;
+import java.io.IOException;
+import java.util.Map;
+import java.util.UUID;
+import org.apache.catalina.session.Session;
+import org.apache.catalina.session.SessionManager;
+import org.apache.coyote.http11.Cookies;
+import org.apache.coyote.http11.Request;
+import org.apache.coyote.http11.Response;
+import org.apache.coyote.util.UrlEncodedFormParser;
+
+public class LoginController extends AbstractController {
+
+    private static final String SESSION_COOKIE_KEY = "JSESSIONID";
+
+    @Override
+    protected void doGet(final Request request, final Response response) throws Exception {
+        if (isUserAlreadyLoggedIn(request)) {
+            response.sendRedirection("/index.html");
+            return;
+        }
+        response.sendResource("/login.html");
+    }
+
+    @Override
+    protected void doPost(final Request request, final Response response) throws Exception {
+        final Map<String, String> formData = UrlEncodedFormParser.parse(request.getMessageBody());
```

Comment:
해당 파싱 로직을 Request 클래스로 이동하는 건 어떤 것 같아요 ❓ 

Request가 생성될 때 자신의 Content-Type에 맞게 Body를 미리 파싱해둔다면 Controller의 책임이 줄어들 것 같고, 자연스러운 흐름이 될 것 같다는 생각이 들어요!

```suggestion
        // 1
        final String account = request.getParameter("account")
        
        // 2
        final Map<String, String> formData = request.getParameters();
```